import { UseCaseError } from "../../../../core/logic/UseCaseError";
import { Result } from "../../../../core/logic/Result";

export namespace GenerateScheduleErrors {

  // export class EmployeeAlreadyExists extends Result<UseCaseError> {    
  //   constructor (name: string) {
  //     super(false, {
  //       message: `The employee ${name} already exists`
  //     } as UseCaseError)
  //   }
  // }

}